docs/token-storage.md

```diff
@@ -0,0 +1,26 @@
+## Token Storage
+
+Once the user has successfully authenticated, you'll have a JWT and possibly a refresh token that need to be stored securely.
+
+❗️ __Do not use Async Storage for storing sensitive information__
+
+Async Storage is the simplest method of persisting data across application launches. However, it is _unencrypted_ key-value store and so should not be used for token storage.
+
+✅ __DO use Secure Storage__
+
+React Native does not come bundled with any way of storing sensitive data, however there are pre-existing solutions for both platforms.
+
+### iOS - Keychain Services
+Keychain Services allows you to securely store small chunks of sensitive info for the user. This is an ideal place to store certificates, tokens, passwords, and any other sensitive information that doesn’t belong in Async Storage.
+
+### Android - Secure Shared Preferences
+Shared Preferences is the Android equivalent for a persistent key-value data store. Data in Shared Preferences is not encrypted by default. Encrypted Shared Preferences wraps the Shared Preferences class for Android, and automatically encrypts keys and values.
+
+In order to use iOS Keychain services or Android Secure Shared Preferences, you either can write a bridge yourself or use a library which wraps them for you and provides a unified API at your own risk.
```

Comment:
```suggestion
In order to use iOS's Keychain services or Android's Secure Shared Preferences, you either can write a JS<->native interface yourself or use a library which wraps them for you. Some even provides a unified API.
```

Which one would you recommend of the 4 libraries you listed below?

Reply:
I'm not officially recommending any of them, but I put the one we like the most at the top 🤷‍♀